src/components/Calendar/SingleDayPicker.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { SingleDatePicker } from 'react-dates';
+import 'react-dates/initialize';
+
+import { CalendarWrap } from './components';
+
+import CloseIcon from './close.svg';
+import ArrowRightIcon from './arrow-right.svg';
+
+const SingleDayPicker = props => (
+  <CalendarWrap>
+    <SingleDatePicker
+      navNext={<ArrowRightIcon />}
+      navPrev={<ArrowRightIcon />}
+      customCloseIcon={<CloseIcon />}
```

Comment:
How do these pickers close? Would you usually provide an `onClose` prop?

Reply:
The three components are exported as *they are*, so they can receive all the props the original components receive. What I've passed here are `healthy defaults` and UI components that can still be overridden if necessary.